src/app/ui/editor/pixels_movement.cpp

```diff
@@ -391,28 +427,9 @@ void PixelsMovement::moveImage(const gfx::PointF& pos, MoveModifier moveModifier
         else
           dy = 0.0;
       }
-
       bounds.offset(dx, dy);
-
-      if ((m_site.tilemapMode() == TilemapMode::Tiles) ||
-          (moveModifier & SnapToGridMovement) == SnapToGridMovement) {
```

Comment:
Why this was removed? It looks like now we cannot snap to grid when moving pixels when we press <kbd>Alt</kbd> key.